### PR TITLE
Connector auth + CIP30 update

### PR DESCRIPTION
**Pull request by zuzunker**

### CODE REVIEW

File: packages/yoroi-extension/app/ergo-connector/stores/ConnectorStore.js [hunk 1]
```diff
@@ -582,6 +580,51 @@ export default class ConnectorStore extends Store<StoresMap, ActionsMap> {
     }
     return result ?? [];
   }
+  static createAuthEntry: ({|
```
Comment (yushih) [inline packages/yoroi-extension/app/ergo-connector/stores/ConnectorStore.js]: Maybe alternatively consider putting these `static` methods in a new file `app/ergo-connector/api/index.js` ?

File: packages/yoroi-ergo-connector/src/inject.js [hunk 2]
```diff
@@ -383,7 +479,12 @@ if (shouldInject()) {
...
                             imgBase64Url,
                             type: `yoroi_connect_request/${protocol}`,
-                            url: location.hostname,
+                            connectParameters: {
+                                url: location.hostname,
+                                requestIdentification: event.data.requestIdentification,
+                                onlySilent: event.data.onlySilent,
+                                protocol,
+                            },
                             protocol,
```
Comment (neuodev) [inline packages/yoroi-ergo-connector/src/inject.js]: The `protcol` is duplicated, please remove one and update the logic

**Review by zuzunker — COMMENTED** [review]
/check

**Review by zuzunker — COMMENTED** [review]
/check

**Review by zuzunker — COMMENTED** [review]
/check

**Pull request merged**